integration/maptest/map.go

```diff
@@ -132,14 +132,19 @@ func verifyGetMapLeavesResponse(getResp *trillian.GetMapLeavesResponse, indexes
 }
 
 // newTreeWithHasher is a test setup helper for creating new trees with a given hasher.
-func newTreeWithHasher(ctx context.Context, tadmin trillian.TrillianAdminClient, hashStrategy trillian.HashStrategy) (*trillian.Tree, hashers.MapHasher, error) {
+func newTreeWithHasher(ctx context.Context, tadmin trillian.TrillianAdminClient, tmap trillian.TrillianMapClient, hashStrategy trillian.HashStrategy) (*trillian.Tree, hashers.MapHasher, error) {
 	treeParams := stestonly.MapTree
 	treeParams.HashStrategy = hashStrategy
 	tree, err := tadmin.CreateTree(ctx, &trillian.CreateTreeRequest{
```

Comment:
This combination of `CreateTree`, `InitMap` could be pulled out into a general `CreateAndInitMap` function